tests/aiperf_mock_server/README.md

```
@@ -416,109 +416,147 @@ Models with `gpt-oss` or `qwen` in the name support reasoning:
-By default, the scheduler simply queues excess decoders past `max_batch_size` and aggregate throughput plateaus. Real continuous-batching servers exhibit *goodput collapse* under heavy oversubscription: preemption thrash, swap, and admission churn drop the *useful* tok/s once the queue grows too large. Enable this with `--scheduler-goodput-collapse-enabled`:
+**Summary** — `<PATH>.summary.json` and stdout, per endpoint:
 
-```bash
-aiperf-mock-server \
-  --scheduler-enabled \
-  --scheduler-max-batch-size 64 \
-  --scheduler-goodput-collapse-enabled \
-  --scheduler-goodput-collapse-threshold 1.5 \
-  --scheduler-goodput-collapse-slope 0.5 \
-  --scheduler-goodput-collapse-floor 0.3
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)